### PR TITLE
Add command line control interface

### PR DESCRIPTION
**Pull request by dhellmann**

### CODE REVIEW

File: akanda/rug/commands.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+"""Constants for the commands
+"""
+
+WORKERS_DEBUG = 'workers-debug'
```
Comment (ryanpetrello) [inline akanda/rug/commands.py:5]: Explain what these commands signify with comments.

Alternatively, should we define the actual command implementations you wrote as worker.py `def _dispatch_command` in here instead?

File: akanda/rug/cli/router.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            'command': self._COMMAND,
+            'router_id': parsed_args.router_id,
+        }
+
+
+class RouterDebug(_RouterCmd):
+    """debug a single router"""
+
+    _COMMAND = commands.ROUTER_DEBUG
+
```
Comment (ryanpetrello) [inline akanda/rug/cli/router.py:36]: Did you consider overriding `take_action` to (in addition to pushing a debug message to workers) subsequently call into `akanda.rug.debug:debug_one_router` (or at least provide the option to do so?)  It seems like you'd want to do this most of the time.

File: akanda/rug/commands.py (hunk 1 above)
Comment (dhellmann) [inline akanda/rug/commands.py:5]: I was a little torn about that. The strings here are part of the API between the client and the worker process. I didn't want the implementation details of the worker to bleed into code used in the client. The actual message structures probably should be defined more here.

Comment (ryanpetrello) [inline akanda/rug/commands.py:5]: Makes sense.  I'd at least like explanations here so you know what these mean.  The start/stop debugging nature of these commands wasn't immediately obvious to me until I read the implementation.

File: akanda/rug/cli/router.py (hunk 2 above)
Comment (dhellmann) [inline akanda/rug/cli/router.py:36]: I can see that being a useful option, but I was concentrating on the message-passing side of things here.

Comment (ryanpetrello) [inline akanda/rug/cli/router.py:36]: Yep, just a thought I had while reading.  I'd be fine with this coming in a later pull request.

File: akanda/rug/commands.py (hunk 1 above)
Comment (dhellmann) [inline akanda/rug/commands.py:5]: Done

**Pull request merged**